### PR TITLE
Refs #5148 - Package page applicability

### PR DESCRIPTION
**Pull request by daviddavis**
Add the ability to view package applicability and upgradability from the packages screen.

- [x] Add tests
- [x] Fix broken tests

### CODE REVIEW

File: app/views/katello/api/v2/packages/base.json.rabl [hunk 1]
```diff
@@ -3,3 +3,6 @@ object @resource
 attributes :id, :uuid, :name, :version, :release, :arch, :epoch, :summary
 attributes :filename, :sourcerpm, :checksum
 attributes :nvrea, :nvra
+
+node(:hosts_available_count) { |m| m.hosts_available(params[:organization_id]).count }
+node(:hosts_applicable_count) { |m| m.hosts_applicable(params[:organization_id]).count }
```
Comment (daviddavis) [inline app/views/katello/api/v2/packages/base.json.rabl]: @jlsherrill wondering if this'll cause a hit to performance? we do it for errata as well.

Comment (jlsherrill) [inline app/views/katello/api/v2/packages/base.json.rabl]: It very well could, but we haven't seen a huge issue with errata, so i guess its okay?

**Comment by daviddavis** [comment]
@jlsherrill I think this should be ready for review if you wanted to review it.

File: app/models/katello/host/content_facet.rb [hunk 2]
```diff
@@ -54,6 +54,15 @@ def installable_errata(env = nil, content_view = nil)
...
       end
 
+      def installable_rpms(env = nil, content_view = nil)
+        repos = if env && content_view
+                  Katello::Repository.in_environment(env).in_content_views([content_view])
+                else
+                  self.bound_repositories.pluck(:id)
+                end
+        self.applicable_rpms.in_repositories(repos).uniq
+      end
```
Comment (jlsherrill) [inline app/models/katello/host/content_facet.rb]: test please!

**Comment by daviddavis** [comment]
@jlsherrill thanks for the review. I've updated the PR.

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**